source/index.ts

```diff
@@ -172,7 +167,7 @@ namespace is { // tslint:disable-line:no-namespace
 	export const truthy = (value: any) => Boolean(value);
```

Comment:
Same as https://github.com/sindresorhus/is/pull/49/files?w=1#r185387543 for `directInstanceOf`'s `klass` argument. (I can't comment on the exact line.